.github-scheduled-workflows/publish-main.yaml

```diff
@@ -30,6 +30,7 @@ jobs:
         uses: actions/checkout@v3
         with: 
            path: operator
+           ref: {{ env.release-version }}
```

Comment:
I think we need to be checking  out the latest `main` here as we need to update that, not the tag associated with the relase.

Reply:
Tag was created from main branch only so it will same as main .
Also it doesn't matter which branch/tag we checkout. The updated csv file will be pushed to targetBranch: 'main' as mentioned in the next step

Reply:
:+1: I missed that `yaml-update-action` was explicitly performing the checkout on main.